src/index.ts

```diff
@@ -22,7 +22,7 @@ export function parseFile(text: string, path: string, isEntry: bool = false,
 }
 
 /** Obtains the next required file's path. Returns `null` once complete. */
-export function nextFile(parser: Parser): string | null {
+export function nextFile(parser: Parser): [string, string] | null {
```

Comment:
Perhaps having an API that doesn't require the caller to deal with arrays would be preferable here (e.g., if somehow linking with C in the future / just strings is considerably easier to do). Like, let's say, a `getDependee` function that obtains the dependee from the `file` returned by `nextFile`, using an internal map.